<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAdmissions</class>
 <widget class="QWidget" name="PageAdmissions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* STYLE LOCAL */
QWidget { font-family: &quot;Segoe UI&quot;, sans-serif; background-color: #f0f2f5; }

/* Cadres */
QGroupBox, QFrame.Card { 
    font-weight: bold; background-color: white; border: 1px solid #ccc; border-radius: 5px; margin-top: 10px; color: #333333; 
}
QGroupBox::title { subcontrol-origin: margin; left: 10px; padding: 0 5px; color: #333333; }

/* Textes */
QLabel { color: #333333; font-size: 14px; }

/* Champs */
QLineEdit, QComboBox, QDateEdit { 
    border: 1px solid #ccc; border-radius: 4px; padding: 6px; background-color: #fafafa; min-height: 25px; color: #333333; 
}
QLineEdit:focus, QComboBox:focus, QDateEdit:focus { border: 1px solid #2d7aed; background-color: white; }

QComboBox QAbstractItemView { background-color: white; color: #333333; selection-background-color: #2d7aed; selection-color: white; outline: 0; }

/* Boutons */
QPushButton.ActionBtn { background-color: #2d7aed; color: white; border: none; border-radius: 4px; padding: 8px 16px; font-weight: bold; }
QPushButton.ActionBtn:hover { background-color: #1a5fc7; }

/* Tableau */
QTableWidget { background-color: white; border: none; gridline-color: #f0f0f0; color: #333333; }
QHeaderView::section { background-color: #f8f9fa; padding: 5px; border: none; border-bottom: 2px solid #2d7aed; font-weight: bold; color: #333333; }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   
   <item>
    <widget class="QGroupBox" name="groupBox_new_admission">
     <property name="title">
      <string>Enregistrer une nouvelle admission</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_patient">
        <property name="text">
         <string>Patient :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_patient">
        <property name="placeholderText">
         <string>Rechercher un patient...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_room">
        <property name="text">
         <string>Chambre disponible :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_room"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_date">
        <property name="text">
         <string>Date d'entrée :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit_entry">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_save">
        <property name="class" stdset="0">
         <string>ActionBtn</string>
        </property>
        <property name="text">
         <string>Valider l'Admission</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <item>
    <widget class="QFrame" name="frame_list">
     <property name="class" stdset="0">
      <string>Card</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_list">
      <item>
       <widget class="QLabel" name="label_list_title">
        <property name="text">
         <string>Hospitalisations en cours</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px; font-weight: bold; margin-bottom: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableAdmissions">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Chambre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Patient</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Entrée le</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>